packages/react-native-bpk-component-rating/src/common-types.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import PropTypes from 'prop-types';
+
+type StringOrArray = string | Array<string>;
+
+const SIZES = {
+  icon: 'icon',
+  xs: 'xs',
+  sm: 'sm',
+  base: 'base',
+  lg: 'lg',
+};
```

Comment:
Why don't we export this and let consumers use it? If we do, we should update the readme sample code too.

Reply:
My issue with exposing those objects and advertising its usage is that it doesn't make much sense now that we use flow. 

When you are typing code and add `size=` flow will auto suggest the list of valid strings to type in, the same way it will show the list of valid options when it shows an error.

Adding this "emun" object only creates indirection, where instead of typing `size="sm"` you have to type `size={SIZES.sm}` with no benefit. The result of a typo using flow will be mostly the same (`sn` is not supported) and without flow I'd argue the result is worse with the enum object, because in this case you will not see an error but instead the default value for the property will be used, because `sn` will be `undefined`.

Reply:
Makes sense to me - thanks for the detailed explanation!